definitions/pipelines/immuno.cwl

```diff
@@ -285,6 +285,9 @@ inputs:
         type: boolean?
     pvacseq_threads:
         type: int?
+    vep_to_table_prefix:
+        type: string?
+        default: 'pvacseq'
```

Comment:
Should this default be in the `pvacseq.cwl`?

Reply:
I was trying to minimize changes outside of the immuno.cwl, but this change can probably go there instead without disrupting anything else